cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -1,17 +1,18 @@
 // Command Group: "Data Conversion"
 
-// Generates a VCF file from an input pairwise multiFa alignment with the first entry as the reference
+// Generates a VCF file from an input pairwise or three-way multiFa alignment with the first entry as the reference
 package main
 
 import (
 	"flag"
 	"fmt"
+	"log"
+
```

Comment:
Check out the file in this directory multiFaToVcf_test.go. Here we'll need to add a test case for when we have 3 sequences. You can also check out the testdata subdirectory to see how tests were set up for the pairwise case and use this as a template